helix-term/src/commands/typed.rs

```diff
@@ -1474,6 +1505,13 @@ pub const TYPABLE_COMMAND_LIST: &[TypableCommand] = &[
             fun: sort_reverse,
             completer: None,
         },
+        TypableCommand {
+            name: "reflow",
+            aliases: &[],
+            doc: "Hard-wrap the current selection of lines to a given width.",
+            fun: reflow,
+            completer: None,
+        },
```

Comment:
I wonder if we should just bind it to `gq`.

Reply:
I don't know any other good bindings for it beside `gq` which is similar to vim, we might have to tweak `goto menu` to `goto/misc menu`

Reply:
Seems like this decision shouldn't block the PR. I imagine it's pretty easy to add an alias later.